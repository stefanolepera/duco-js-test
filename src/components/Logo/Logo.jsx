import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
    margin: 30px auto;
`;

const Logo = () => (
    <Wrapper>
        <img
            src="https://cssanimation.rocks/demo/starwars/images/star.svg"
            alt="Star"
        />
        <img
            src="https://cssanimation.rocks/demo/starwars/images/wars.svg"
            alt="Wars"
        />
    </Wrapper>
);

export default Logo;
