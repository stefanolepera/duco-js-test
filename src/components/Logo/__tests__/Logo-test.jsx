import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Logo from '../Logo';

describe('<Logo /> test', () => {
    it('Test snapshots of content', () => {
        const wrapper = shallow(<Logo />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
