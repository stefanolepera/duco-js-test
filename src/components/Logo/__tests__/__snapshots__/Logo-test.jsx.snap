// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> test Test snapshots of content 1`] = `
<styled.div>
  <img
    alt="Star"
    src="https://cssanimation.rocks/demo/starwars/images/star.svg"
  />
  <img
    alt="Wars"
    src="https://cssanimation.rocks/demo/starwars/images/wars.svg"
  />
</styled.div>
`;
